src/main/java/org/schemaspy/Config.java

```diff
@@ -626,6 +628,33 @@ public int getFontSize() {
 
         return fontSize;
     }
+    
+    /**
+     * @return
+     * @see #setRelationDegreeDepth(int)
+     */
+    public int getRelationDegreeDepth() {
```

Comment:
Would it be possible to move this to cli.CommandLineArguments

We actually want to remove the Config class.

It's not actually a big thing, since I personally think that we should switch from JCommander to Picocli since it's maintained. But that a whole other story.